care/facility/api/serializers/file_upload.py

```diff
@@ -123,6 +123,14 @@ class Meta:
         )
         write_only_fields = ("associating_id",)
 
+    def validate(self, attrs):
+        user = self.context["request"].user
+        if user.user_type < User.TYPE_VALUE_MAP["Nurse"]:
+            raise serializers.ValidationError(
+                {"permission": "Only Nurses and above can upload files."}
+            )
+        return super().validate(attrs)
```

Comment:
Why are Authz checks done in the serializer validate? Shouldn't this be done by DRY Permissions?